src/styles/index.scss

```diff
@@ -1,3 +1,147 @@
+$main-color: #060b35;
+$secondary-color: #616070;
+$bg-color: #00acdc;
+$border-color: #f3f3f3;
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+}
+
 body {
-  margin: 0;
+  margin: 0 auto;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.card {
+  &__container {
+    margin: 0 auto;
+    box-sizing: border-box;
+    padding: 36px 16px 16px;
+    width: 200px;
+    height: 408px;
+    background-color: #fff;
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: start;
+  }
+  &__image {
+    box-sizing: border-box;
+    margin-bottom: 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-top: 0;
+    margin-bottom: 4px;
+
+    text-align: start;
+    text-wrap: nowrap;
```

Comment:
The `text-wrap` property is not a valid CSS property. You might want to use `white-space: nowrap;` if you intend to prevent text from wrapping.